assets/js/googlesitekit-activation.js

```diff
@@ -41,7 +42,10 @@ domReady( () => {
 		trackEvent( 'plugin_setup', 'plugin_activated' );
 
 		render(
-			<Root>
+			<Root
+				dataAPIContext="Activation"
```

Comment:
Since the data API is now removed (apologies for the long review delay here), this should no longer be needed. Can you please remove usage of this prop throughout the PR?